page-container/index-page/ContactForm.tsx

```diff
@@ -0,0 +1,152 @@
+import {
+  Typography, Grid, makeStyles, TextField, Button
+} from '@material-ui/core';
+import { useCallback, useState } from 'react';
+import packageConfig from '../../package.json';
+
+const useStyle = makeStyles((theme) => ({
+  title: {
+    color: '#ffffff'
+  },
+  formRoot: {
+    marginTop: '7rem',
+    width: '28.75rem',
+    '& > div': {
+      marginBottom: '3rem',
+      '&:last-child': {
+        marginTop: '1.8125rem',
+        marginBottom: 0
+      }
+    },
+    '& input, & textarea, & label': {
+      fontSize: '1.125rem',
+      color: theme.palette.text.secondary
+    },
+    '& button[type="submit"]': {
+      backgroundColor: theme.palette.background.paper,
+      borderRadius: '3px'
+    }
+  },
+  inputRoot: {
+    '& > .MuiInput-underline:before': {
+      borderBottomColor: '#333333',
+      bottom: '-0.25rem'
+    },
+    '& textarea::placeholder': {
+      color: 'white'
+    }
+  }
+}));
+
+type FormState = {
+  name: string;
+  email: string;
+  phone: string;
+  company: string;
+  message: string;
+} | null;
+
+const formPostUrl = packageConfig.pedialab.contactFormEndpoint;
+
+const ContactForm = ({ className }: Partial<{ className: string }>) => {
+  const classes = useStyle();
+
+  const [formState, setFormState] = useState<FormState>(null);
+
+  const handleSubmit = useCallback(
+    (event: React.FormEvent) => {
+      event.preventDefault();
+
+      if (formState) {
+        fetch(formPostUrl, {
```

Comment:
although this is a very simple request, suggest always to extract these app logic into service layer / state container layer to help testing